app/controllers/api/v1/messages_controller.rb

```diff
@@ -0,0 +1,63 @@
+module Api
+  module V1
+    class MessagesController < ApplicationController
+      before_action :set_message, only: %i[show update destroy]
+
+      # GET /messages
+      def index
+        @messages = Message.all
+
+        render json: @messages
+      end
+
+      # GET /messages/1
+      def show
+        render json: @message
+      end
+
+      # GET /message/random
+      def random
+        @random_greeting = Message.order('RANDOM()').first!
+
+        render json: @random_greeting
+      end
```

Comment:
- [ ] Just like the previous reviewer suggested, please ensure that the React app displays a random message. Kindly see error below 👇 
![image](https://user-images.githubusercontent.com/46839548/208242029-badec412-d54a-4943-933b-adcf8a2f4da5.png)
The proxy which you set in your package.json file doesn't seem to be working. Maybe you could try passing in the correct URL to the fetch API manually if you can't find a workaround 👍